roles/confluent.schema_registry/tasks/main.yml

```diff
@@ -116,29 +116,12 @@
 
 - name: Create Logs Directory
   file:
-    path: "{{schema_registry.appender_log_path}}"
+    path: "{{schema_registry_log_dir}}"
     state: directory
     group: "{{schema_registry_group}}"
     owner: "{{schema_registry_user}}"
     mode: 0770
 
-- name: Create log4j Directory
```

Comment:
What happens if the user changes the default logging location, starts the process, then decides to change the user and restart. By not applying the user and group ownership, does this still work?